src/main/webapp/WEB-INF/pages/contact.ftl

```diff
@@ -1,35 +1,28 @@
-<div id="contact" class="container-fluid">
-    <div class="cont-style">
-        <br>
-        <h2 class="text-center">CONTACT</h2>
-        <br>
-        <div class="row">
-            <div class="col-sm-5">
-                <p>Contact us and we'll get back to you within 24 hours.</p>
-                <p><span class="glyphicon glyphicon-map-marker"></span> Cluj-Napoca, RO</p>
-                <p><span class="glyphicon glyphicon-phone"></span> 0040 747123123</p>
-                <p><span class="glyphicon glyphicon-envelope"></span> office@markettime.com</p>      
-            </div>
-            <form name="contact" action="contact" method="POST">
-	           <div class="col-sm-7 slideanim">
-	               <div class="row">
-	                   <div class="col-sm-6 form-group">
-	                       <input class="form-control" id="companyName" name="companyName" placeholder="Company name" type="text" required>
-	                   </div>
-    	               <div class="col-sm-6 form-group">
-    	                   <input class="form-control" id="email" name="email" placeholder="Email address" type="email" required>
-    	               </div>
-	               </div>
-	           </div>
-	           <textarea class="form-control" id="comments" name="comments" placeholder="Comment" rows="5"></textarea><br>
-	           <div class="row">
-	               <div class="col-sm-12 form-group">
-	                   <button class="btn btn-default pull-left" type="submit">Send</button>
-	               </div>
-	           </div>    
-	       </div>
+<div id="contact" class="container-fluid con-container">
```

Comment:
in general, foloseste nume intregi pentru clasele css.
in cazul de fata, nu e nevoie de o clasa noua daca vrei sa o aplici pe un singur element; pentru asta poate fi folosit id-ul elementului.